src/System.Runtime/tests/System/ArrayOfPointers.ArrayTests.cs

```diff
@@ -0,0 +1,68 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using Xunit;
+
+namespace System.Tests
+{
+    public static partial class ArrayTests
```

Comment:
This one should not be temporary. If array of pointers work everywhere else, I do not see why they cannot work in ProjectN as well. There is no fundamental limitation that should prevent them from working.